website/docs/table-group/data-table/data-table-a11y.md

```diff
@@ -3,20 +3,55 @@ title: DataTable
 tabs: Design('data-table'), Example('data-table-code'), API('data-table-api'), A11y('data-table-a11y'),  Changelog('data-table-changelog')
 ---
 
+## What component has
+
+### Keyboard support
+
+Table: Keyboard support
+
+| Key                                                   | Function                                              |
+| ----------------------------------------------------- | ----------------------------------------------------- |
+| `Tab`                                                 | Moves focus to the next focusable element.            |
+| `Shift + Tab`                                         | Moves focus to the previous focusable element.        |
+| `Left Arrow`, `Right Arrow`, `Up Arrow`, `Down Arrow` | Move the focus across the grid of table cells.        |
+| `Enter`                                               | Activates interactive elements inside the table grid. |
+
+### Roles & attributes
+
+The following list describes roles and attributes that component already has.
+
+| Component / element               | Role / Attribute                        | Usage                                                                                                                                                                                                                                                                                                             |
+| --------------------------------- | --------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `DataTable`                       | `grid`                                  | Identifies the element that serves as the grid widget container. The `grid` element isn't focusable because it implements the practice described in the section on [Managing Focus Within Components Using a Roving tabindex](https://www.w3.org/WAI/ARIA/apg/practices/keyboard-interface/#kbd_roving_tabindex). |
+|                                   | `aria-rowcount`                         | Defines the total number of rows in the `grid`.                                                                                                                                                                                                                                                                   |
```

Comment:
shouldn't this and colcount both be defined only optionally, when the visible number of rows/cols is different from DOM? i.e. shouldn't these 2 attributes be in the "Considerations" section?

Reply:
That's a good question. Let me check and I'll update this

Reply:
Actually, I'm not sure they need to be in the Considerations section, since it usually contains attributes teams need to consider and specify themselves.

Reply:
but that was exactly my point, rowcount and colcount attributes are not mandatory and should be defined only in specific cases, don't they?

Reply:
I'd just add a note about certain conditions for these attributes, but I wouldn't move them to the Considerations section.

Reply:
Updated a little bit description for `aria-rowcount` and `aria-colcount`, `aria-rowindex` and `aria-colindex`. They all have kinda general description of how they work. And the teams do not need to set them themselves, they are set by our component.